spec/invoice_repository_spec.rb

```diff
@@ -0,0 +1,102 @@
+require_relative 'spec_helper'
+require_relative '../lib/invoice_repository'
+
+RSpec.describe InvoiceRepository do
+  before :each do
+    @repo = InvoiceRepository.new('./spec/fixtures/mock_invoices.csv')
+  end
+
+  it 'exists' do
+    expect(@repo).to be_an_instance_of(InvoiceRepository)
+  end
+
+  it 'returns all merchants' do
+    expect(@repo.all.length).to eq(3)
```

Comment:
Could be good to add the `.each` here in and iterate through `.all` in order to test to see if each invoice is an instance of Invoice.